### PR TITLE
feat: add bug report template

### PR DESCRIPTION
**Pull request by castrojo**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,38 +1,65 @@
...
+        - Mate
+        - Silverblue
+        - Ubuntu
+        - Vauxite
+    validations:
+      required: true
+  - type: textarea
+    id: status
+    attributes:
+      label: Cloud Custodian version and dependency information
```
Comment (lazypower) [inline .github/ISSUE_TEMPLATE/bug_report.md:46]: is CC supposed to be here or is this copy/paste?

**Review by lazypower — COMMENTED** [review]
One question left inline, otherwise LGTM

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (castrojo) [reply .github/ISSUE_TEMPLATE/bug_report.md:46]: pasta, will fix.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,38 +1,65 @@
...
+      description: Please run `rpm-ostree status -v` and paste the output here.
+      render: shell
+  - type: textarea
+    id: policy
+    attributes:
+      label: Policy
+      description: Please copy and paste the policy you are trying to run if applicable. Please exclude any account/sensitive information. Your response will be automatically formatted into YAML, so code-formatting markdown is not necessary.
+      render: shell
```
Comment (marcoceppi) [inline .github/ISSUE_TEMPLATE/bug_report.md:54]: What is a policy?

**Review by marcoceppi — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (castrojo) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thought I removed all this, sorry late nite last night.

**Review by joshua-stone — APPROVED** [review]
LGTM

**Pull request merged**